File: Modul-4/C/PRAK405-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
  int n, m;
  int total = 0;

  printf("Input\n");
  scanf("%d %d", &n, &m);

  printf("Output\n");
  for (int i = 1; i <= n; i++)
  {
    int hasil = 0;
    for (int j = i; j >= 1; j--)
    {
      printf("(%d * %d)", j, m);
      if (j > 1)
      {
        printf(" + ");
      }
      hasil += j * m;
    }
    total += hasil;
    printf(" = %d\n", hasil);
  }

  printf("%d\n", total);

  return 0;
}

File: Modul-4/Python/PRAK405-2310817110006-Adrian_Bintang_Saputera.py
n, m = map(int, input("Input\n").split())
total = 0

print("Output")
for i in range(1, n+1):
    hasil = 0
    for j in range(i, 0, -1):
        print(f"({j} * {m})", end="")
        if j > 1:
            print(" + ", end="")
        hasil += j * m
    total += hasil
    print(f" = {hasil}")

print(total)